Exercises_7_a_b_c.cpp

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
I like comments at the top of every file that tell what it is:
// Answers to exercise 7(a)(b) and (c)